gulpfile.js

```diff
@@ -1,10 +1,10 @@
-var gulp = require('gulp'),
+let gulp = require('gulp'),
```

Comment:
Excellent questions.  I believe it still works but I'll test it.  As for import, I've never really seen import being used in gulpfiles before.  I prefer the import syntax over the require syntax so if we decide that's how we want to do it, then you'll hear no objections from me.